.github/workflows/release.yml

```
@@ -0,0 +1,58 @@
+          REPO_NAME="${GITHUB_REPOSITORY#*/}"
+
+          echo "TAG=${TAG}" | tee -a ${GITHUB_ENV}
+          echo "SEMANTIC_TAG=${TAG#v}" | tee -a ${GITHUB_ENV}
+          echo "REPO_NAME=${REPO_NAME}" | tee -a ${GITHUB_ENV}
+          echo "IMAGE_PATH=quay.io/open-cluster-management/${REPO_NAME}:${TAG}" | tee -a ${GITHUB_ENV}
+
+      - name: Checkout code
+        uses: actions/checkout@v6
+
+      - name: Package Helm chart
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:6:82: Double quote to prevent globbing and word splitting

(shellcheck)